test/test.js

```diff
@@ -208,6 +208,24 @@ describe( 'rollup-plugin-typescript', function () {
 			sourceMap: false
 		});
 	});
+	
+	it( 'does not include helpers in source maps', function () {
+		return bundle( 'sample/dedup-helpers/main.ts', {
+			sourceMap: true
+		}).then( function ( bundle ) {
+			const result = bundle.generate({
+				sourceMap: true
+			});
+			
+			const sources = result.map.sources;     
+			
+			function isNotHelper( path ) {
+				return path.indexOf( 'typescript-helpers' ) === -1;
```

Comment:
I was just checking that no source path in the generated source map includes the word `typescript-helpers`. That should cover both `\0typescript-helpers` and the path to `typescript-helpers.js`. It's not great, but gets the job done in this simple example.